shanghai/core_plugins/channel.py

```diff
@@ -286,7 +359,7 @@ def chan_eq(ctx: NetworkContext, chan1: str, chan2: str):
     return ctx.chan_lower(chan1) == ctx.chan_lower(chan2)
 
 
-@global_event(GlobalEventName.INIT_NETWORK_CTX)
+@global_event(GlobalEventName.INIT_NETWORK_CTX, priority=Priority.POST_CORE)
```

Comment:
It needs to run before "DEFAULT" (so normal plugins can use this) and after "PRE_CORE" (message.py).